src/modules/storefront/provider.tsx

```diff
@@ -39,11 +31,11 @@ export const StorefrontProvider = ({ wallet, children }: StorefrontProviderProps
       setStorefront(storefront);
       setSearching(false);
     });
-  }, [wallet?.pubkey]);
+  }, [pubkey]);
 
   return (
     <StorefrontContext.Provider value={{ searching, storefront }}>
-      {children({ searching, storefront })}
+      {children}
```

Comment:
If taking above suggestion, then:

```suggestion
      {props.children}
```